app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,19 @@
 
     <p class="notice"><%= notice %></p>
     <p class="alert"><%= alert %></p>
-
+materialize
```

Comment:
Please fix this code, I think you had conflicts and you didn't solve it when merging or you solve them but in the wrong way, I'm pretty sure this `materialize` text doesn't need to be here, also the `navbar-text` class belongs to the header isn't it?